spec/requests/service_plans_spec.rb

```diff
@@ -117,7 +119,8 @@
 
     context "when there is not a modified schema" do
       before do
-        service_plan.update!(:modified => nil)
+        service_plan.modified = nil
+        service_plan.save!(:validate => false)
```

Comment:
Why do we need the `:validate => false` here?

Reply:
It was before I forgot to add the `:if => proc { modified.present? }` that we talked about, fixed now!